src/app/app.server.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { ServerModule } from '@angular/platform-server';
+
+import { AppModule } from './app.module';
+import { AppComponent } from './app.component';
```

Comment:
Since we've been using absolute imports for our files, you can also do the same here for consistency.

Reply:
Thanks @wangonya ,i have worked on this